app/models.py

```diff
@@ -137,6 +137,15 @@ class PhoneNumberVerification(db.Model):
     country_code = db.Column(db.String(10))
     national_format = db.Column(db.String(20))
     phone_number = db.Column(db.String(30))
+    def from_twilio_response(input):
+        return PhoneNumberVerification(
+            caller_name = input['caller_name']['caller_name'],
+            caller_type = input['caller_name']['caller_type'],
+            error_code = input['caller_name']['error_code'],
```

Comment:
I think we're going to need to account for `input['caller_name']` sometimes being `null`.

https://www.twilio.com/docs/lookup/api#lookups-caller-name

![image](https://user-images.githubusercontent.com/3099999/109515923-41450b80-7a6d-11eb-8cb5-b3002aeb08cb.png)

I think this is what you want. I'd add a test for when each of these fields is `null` to make sure we don't blow up when those values come back from the API

```suggestion
caller_info = input['caller_name'] or { 'caller_name': None, 'caller_type': None, 'error_code': None }

return PhoneNumberVerification(
    caller_name = caller_info['caller_name'],
    caller_type = caller_info['caller_type'],
    error_code = caller_info['error_code'],
```